### ISSUE TITLE
v2.0.2 wheel corrupt: multiple .dist-info directories

### ISSUE THREAD

**Issue by fphammerle** [WanzenBug/pylint-import-requirements#22]
```python
$ pip3 install pylint_import_requirements-.2-py3-none-any.whl 
Processing ./pylint_import_requirements-2.0.2-py3-none-any.whl
Requirement already satisfied: importlib-metadata in ./lib/python3.6/site-packages (from pylint-import-requirements==2.0.2)
Requirement already satisfied: pylint in ./lib/python3.6/site-packages (from pylint-import-requirements==2.0.2)
Requirement already satisfied: isort in ./lib/python3.6/site-packages (from pylint-import-requirements==2.0.2)
Requirement already satisfied: setuptools in ./lib/python3.6/site-packages (from pylint-import-requirements==2.0.2)
Requirement already satisfied: astroid in ./lib/python3.6/site-packages (from pylint-import-requirements==2.0.2)
Requirement already satisfied: zipp>=0.5 in ./lib/python3.6/site-packages (from importlib-metadata->pylint-import-requirements==2.0.2)
Requirement already satisfied: mccabe<0.7,>=0.6 in ./lib/python3.6/site-packages (from pylint->pylint-import-requirements==2.0.2)
Requirement already satisfied: toml>=0.7.1 in ./lib/python3.6/site-packages (from pylint->pylint-import-requirements==2.0.2)
Requirement already satisfied: wrapt~=1.11 in ./lib/python3.6/site-packages (from astroid->pylint-import-requirements==2.0.2)
Requirement already satisfied: lazy-object-proxy==1.4.* in ./lib/python3.6/site-packages (from astroid->pylint-import-requirements==2.0.2)
Requirement already satisfied: six~=1.12 in ./lib/python3.6/site-packages (from astroid->pylint-import-requirements==2.0.2)
Requirement already satisfied: typed-ast<1.5,>=1.4.0; implementation_name == "cpython" and python_version < "3.8" in ./lib/python3.6/site-packages (from astroid->pylint-import-requirements==2.0.2)
Installing collected packages: pylint-import-requirements
Exception:
Traceback (most recent call last):
  File "/tmp/test/lib/python3.6/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/tmp/test/lib/python3.6/site-packages/pip/commands/install.py", line 360, in run
    prefix=options.prefix_path,
  File "/tmp/test/lib/python3.6/site-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/tmp/test/lib/python3.6/site-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/tmp/test/lib/python3.6/site-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/tmp/test/lib/python3.6/site-packages/pip/wheel.py", line 345, in move_wheel_files
    clobber(source, lib_dir, True)
  File "/tmp/test/lib/python3.6/site-packages/pip/wheel.py", line 305, in clobber
    ', '.join(info_dir))
AssertionError: Multiple .dist-info directories: /tmp/test/lib/python3.6/site-packages/pylint_import_requirements-2.0.1.dist-info, /tmp/test/lib/python3.6/site-packages/pylint_import_requirements-2.0.2.dist-info

```

**Comment by fphammerle** [comment]
for reference:
> PyPI does not allow for a filename to be reused, even once a project has been deleted and recreated.
https://pypi.org/help/

so we'll need a new release `v2.0.3` to fix this

**Comment by fphammerle** [comment]
```sh
$ atool --list pylint_import_requirements-2.0.2-py3-none-any.whl
atool: pylint_import_requirements-2.0.2-py3-none-any.whl: format not known, identifying using file
atool: pylint_import_requirements-2.0.2-py3-none-any.whl: format is `zip'
Archive:  pylint_import_requirements-2.0.2-py3-none-any.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
    14604  2020-06-20 12:00   pylint_import_requirements/__init__.py
     1799  2020-04-16 13:53   pylint_import_requirements-2.0.1-py3.6.egg-info/PKG-INFO
      586  2020-04-16 13:53   pylint_import_requirements-2.0.1-py3.6.egg-info/SOURCES.txt
        1  2020-04-16 13:53   pylint_import_requirements-2.0.1-py3.6.egg-info/dependency_links.txt
       51  2020-04-16 13:53   pylint_import_requirements-2.0.1-py3.6.egg-info/requires.txt
       27  2020-04-16 13:53   pylint_import_requirements-2.0.1-py3.6.egg-info/top_level.txt
        1  2020-04-16 13:53   pylint_import_requirements-2.0.1.dist-info/dependency_links.txt
       27  2020-04-16 13:53   pylint_import_requirements-2.0.1.dist-info/top_level.txt
      709  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/DESCRIPTION.rst
     1237  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/metadata.json
       27  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/top_level.txt
       92  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/WHEEL
     1778  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/METADATA
     1520  2020-06-20 12:12   pylint_import_requirements-2.0.2.dist-info/RECORD
---------                     -------
    22459                     14 files
```

**Comment by WanzenBug** [comment]
Fixed packages released as v2.0.3

**Comment by fphammerle** [comment]
`pipenv lock` now completed successfully

thanks!
